ssl/quic/quic_tls.c

```diff
@@ -856,6 +856,12 @@ int ossl_quic_tls_tick(QUIC_TLS *qtls)
 }
 #endif
 
+int ossl_quic_tls_clear(QUIC_TLS *qtls)
+{
+    qtls->local_transport_params_consumed = 0;
+    return 1;
+}
```

Comment:
You need to check `qtls != NULL`. I would also make this a void function as this is not going to ever fail.

Reply:
Isn't a check for `qtls != NULL` a valid time for this function to fail, in which case we'd return 0?